apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/api/CompilationUnit.kt

```diff
@@ -17,6 +17,12 @@ interface CompilationUnit {
   val outputDir: Provider<Directory>
   val transformedQueriesDir: Provider<Directory>
 
+  fun configure(configure: Action<Params>)
```

Comment:
How is that going to be working from kotlin build scripts ? Can kotlin "recognize" the `configure` or does it have to be called explicitely ?

Reply:
Both in Groovy and Kotlin, this needs to be called explicitly. I figured, `compilationUnit.all` is a callback with possibly multiple units. People may want to do it conditionally.